src/content/docs/en/pages/guides/marketplace/integrations/signed-cookies.mdx

```diff
@@ -44,6 +70,22 @@ Go to the [building section](/en/documentation/products/guides/build/build-an-ap
 
 To configure your integration, proceed as follows:
 
+<Tabs client:visible>
+    <Fragment slot="tab.console">Console</Fragment>
+    <Fragment slot="tab.rtm">Real-Time Manager</Fragment>
+
+<Fragment slot="panel.console">
+1. On the upper-left corner, select **Products menu** > **Edge Application** on the **Build** section.
+2. On the listing page of your edge applications, select the one you want to use with the **Signed Cookies** integration.
+3. On the **Main Settings** tab, under **Modules**, locate the **Edge Functions** switch and turn it on to enable **Functions** on your edge application.
+4. Still under the **Modules** section, locate the **Application Accelerator** switch and turn it on to enable the **Forward Cookies** functionality for your application.
+  - This will be addressed later at the [Rules Engine section](#configuring-a-rule-on-rules-engine).
+4. Click the **Save** button.
+
+You'll receive a successful message indicating your edge application was updated.
```

Comment:
You'll receive a success message indicating that your Edge application has been updated.